### PR TITLE
Adopt build system to work with Raspberrypi and (MariaDB)

### PR DESCRIPTION
**Pull request by Kicer86**

### CODE REVIEW

**Comment by wiejakp** [comment]
@Kicer86  Is this needed? I did build verlihub on raspberry pi many times without issue. What is reasoning behind this?

File: INSTALL [hunk 1]
```diff
@@ -34,6 +34,10 @@ sudo apt install libpcre3-dev libssl-dev mysql-server mysql-client libmysqlclien
 
  sudo apt install python2.7-dev liblua5.2-dev libperl-dev
 
+* Install dependencies on Raspberry Pi OS:
+
+sudo apt install libpcre3-dev libssl-dev mariadb-server mariadb-client libmariadb-dev libmariadb-dev-compat libmaxminddb-dev libicu-dev gettext libasprintf-dev liblua5.2-dev g++ make cmake
```
Comment (wiejakp) [inline INSTALL:39]: 👍🏼 except installing mariadb stuff imho

**Comment by Kicer86** [comment]
> @Kicer86 Is this needed? I did build verlihub on raspberry pi many times without issue. What is reasoning behind this?

yeah, my bad, I probably had some other issues which lead me think I needed this stuff.

**Comment by RoLex** [comment]
There is a short note on this page > https://github.com/Verlihub/verlihub/wiki/Installation#debian-based-distributions

``If you would like to use MariaDB instead:

sudo apt install mariadb-server libmariadb-dev mariadb-client``

File: INSTALL (hunk 1 above)
Comment (RoLex) [reply INSTALL:39]: There is a short note on this page > https://github.com/Verlihub/verlihub/wiki/Installation#debian-based-distributions

````If you would like to use MariaDB instead:
sudo apt install mariadb-server libmariadb-dev mariadb-client````

**Pull request closed without merging**